frontend-cert/css-projects/progress-bar/user-stories.md

```diff
@@ -0,0 +1,15 @@
+1. You should have a `div` with an `h2` heading of `Car Acceleration`.
+
+1. Inside the main div, there should be another `div` with the class `progress-bar` that contains the full width of the progress bar.
+
+1. Your progress bar should be `800px` wide and `25px` high. 
+   
+1. Inside the `progress-bar` `div`, there should be a `span` with the class `percentage` that represents the actual progress of the bar. Set the appropriate display type to the `span` class to adapt it to the parent.
+
+1. Add a background color to the progress percentage to make it visible.
```

Comment:
For consistency, we should stick with the "you should" or "it should" terminology

```suggestion
1. You should have a background color for the progress percentage to make it visible.
```